libs/application/templates/university/src/fields/Review/OtherDocumentsReview.tsx

```
@@ -3,27 +3,76 @@ import { FC } from 'react'
+              const item = otherDocuments[index]
+              if (item.attachments && item.attachments.length > 0) {
+                return (
+                  <>
+                    <Text>{field.externalKey}:</Text>
+                    {item.attachments.map((file) => {
+                      return (
+                        <Box
+                          display="flex"
+                          alignItems="center"
+                          marginBottom="smallGutter"
+                        >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)